### ISSUE TITLE
Service, sensors or configuration UI not visible after install and restart

### ISSUE THREAD

**Issue by saaste** [azogue/fasthue#10]
I'm running Home Assistant 0.114.4 with Hue integration. I installed Fasthue 1.1.1 via HACS. I have no errors in the logs, but even after multiple restarts I'm not seeing the `fasthue.set_scan_interval` service, sensors or configuration UI.

The default Hue integration works just fine. I can control the lights and dimmers work too (with 5 seconds interval)

Any ideas what could be wrong and what could I do to investigate this issue more?

**Comment by djknight90** [comment]
I have the same issue - interested to hear if there is a solution to this

**Comment by saaste** [comment]
I happened to figure this out by an accident. After installing Fasthue via HACS, you also need to install it via Home Assistant integrations.

Just go to Configuration / Integrations, click the `+` icon and search for `Hue`.  You should see an integration called `Hue fast polling custom integration`. Install that and all entities and the service appear.

IMO this step is not very clearly explained in the installation instructions. This is what it says under **Configuration**:
> Add it from the Integrations menu, set the desired new update interval, and you're good to go.

It does say "Add it from the Integrations menu" but somehow I got confused because it wasn't under **Installation** section. Perhaps this step could be clarified a bit? ☺️

**Comment by djknight90** [comment]
I missed this entirely too! Thanks for adding this, saved me a lot of digging around.

**Comment by skleenyhoffer** [comment]
Anybody know where I can get instructions that actually help? This doesn't work.

"Add it from the Integrations menu, set the desired new update interval, and you're good to go." -nope. Not good to go. Confusing.
"You should see an integration called Hue fast polling custom integration." -nope. Don't see that. There's an integration under HACS for Fast-Hue polling not under Configuration>Integrations. 
![image](https://user-images.githubusercontent.com/65827014/98993525-9845c580-24fc-11eb-9cbd-5a6ed6b938dc.png)

**Comment by saaste** [comment]
@skleenyhoffer I don't know why you added a screenshot of Resource Manager, but I'll do my best to clarify how to make this easier to understand.

1. Install Fasthue either via HACS or copying files manually to `custom_components` folder. I recommend using [HACS](https://hacs.xyz/docs/installation/prerequisites) because it makes installing and updating addons a breeze.
2. Install Fashue integrations by going to `Configuration / Integrations`
![Screenshot 2020-11-13 at 7 45 21](https://user-images.githubusercontent.com/14893875/99034273-7d6b6380-2585-11eb-93b4-fca5cc80a79a.png)
3. Make sure you are on the `Integrations` tab and click the `+` button
![Screenshot 2020-11-13 at 7 46 29](https://user-images.githubusercontent.com/14893875/99034342-9b38c880-2585-11eb-87d0-3177f00f1b71.png)
4. From the opening dialog, search for `fast` and you should see `Hue fast polling custom integration`. Install it.
![Screenshot 2020-11-13 at 7 47 13](https://user-images.githubusercontent.com/14893875/99034439-cfac8480-2585-11eb-8569-13bc87085771.png)

If this does not work, I assume your Fasthue is not installed correctly. `custom_components` folder should be in the same folder as `configuration.yaml`.

**Comment by skleenyhoffer** [comment]
This is exactly what I did but I removed it all. Followed your 
instructions. Still doesn't work. I add Fast-Hue integration from HACS, 
restart and it's NOT under Configuration>Integrations. This is a brand 
new install of Home Assistant with only Hue installed on it. That's the 
only reason I'm using Home Assistant in this location. Fast-hue polling 
doesn't work. Not sure what else to do. The delay for the motion sensor 
currently is unacceptable.



Mika Hiltunen wrote on 11/13/2020 1:03 AM:
